lib/ddtrace/contrib/pg/patcher.rb

```diff
@@ -15,13 +15,7 @@ def patched?
         end
 
         def patch
-          do_once(:pg) do
-            begin
-              patch_pg_client
-            rescue StandardError => e
-              Datadog::Tracer.log.error("Unable to apply pg integration: #{e}")
```

Comment:
No they basically do their own rescue somewhere else. It looks like a bunch of the `patcher` files were changed to prop exception handling up to the framework instead of inside individual contrib packages